README.md

```diff
@@ -1,53 +1,48 @@
 # 🥧 PIE (PHP Installer for Extensions)
 
-You will need PHP 8.1 or newer to run PIE, but PIE can install an extension to any installed PHP version.
+You will need PHP 8.1 or newer to run PIE, but PIE can install an extension to
+any installed PHP version.
 
-## Installing
+If you are an extension maintainer wanting to add PIE support to your extension,
+please read [extension-maintainers](./docs/extension-maintainers.md).
 
-## Usage
+## Installing an extension using PIE
 
-You can download an extension ready to be built or installed using the `download` command. For example, to download the
-`example_pie_extension` extension, you would run:
+You can install an extension using the `install` command. For example, to
+install the `example_pie_extension` extension, you would run:
 
 ```shell
-$ bin/pie download asgrim/example-pie-extension
-You are running PHP 8.3.7
-Target PHP installation: 8.3.7 nts, on Linux/OSX/etc x86_64 (from /usr/bin/php8.3)
+$ bin/pie install asgrim/example-pie-extension
+This command may need elevated privileges, and may prompt you for your password.
+You are running PHP 8.3.10
+Target PHP installation: 8.3.10 nts, on Linux/OSX/etc x86_64 (from /usr/bin/php8.3)
 Found package: asgrim/example-pie-extension:1.0.1 which provides ext-example_pie_extension
-Extracted asgrim/example-pie-extension:1.0.1 source to: /tmp/pie_downloader_6645f07a28bec9.66045489/asgrim-example-pie-extension-769f906
-$ 
+phpize complete.
+Configure complete.
+Build complete: /tmp/pie_downloader_66e0b1de73cdb6.04069773/asgrim-example-pie-extension-769f906/modules/example_pie_extension.so
+Install complete: /usr/lib/php/20230831/example_pie_extension.so
+You must now add "extension=example_pie_extension" to your php.ini
```

Comment:
I thought the idea was to drop configuration files in the `{config_dir}/conf.d/` directory automatically?

Reply:
Yes, there is an issue #20 to be addressed to add this functionality :+1: